### DISCUSSION TITLE
更灵活的用户输入预处理 inputTemplate

### DISCUSSION THREAD

**Discussion by yangchuang** [lobehub/lobehub#969] [General]
### 🥰 需求描述 | Feature Description

类似于DIFY，可以设置更多的变量，变量类型可以为“文本”，“文本域”，“下拉框”等。

<img width="1778" alt="截屏2023-12-14 10 36 05" src="https://github.com/lobehub/lobe-chat/assets/3096747/3b7cd2bd-7a33-41a2-b17a-ed3e391e19a8">
<img width="1090" alt="截屏2023-12-14 11 13 52" src="https://github.com/lobehub/lobe-chat/assets/3096747/ac21c939-5bf4-41ba-9b21-1279a037dc6c">

### 🧐 解决方案 | Proposed Solution

类似于DIFY，可以设置更多的变量，变量类型可以为“文本”，“文本域”，“下拉框”等。

**Comment by arvinxx** [comment]
dify 采用 lexical 实现高级的 prompt 编辑器：

 https://konstantinmuenster.medium.com/how-to-build-a-text-editor-with-lexical-and-react-27204c186d0f
